pyproject.toml

```diff
@@ -18,7 +18,7 @@ dependencies = [
     "inscriptis < 3",
     "jwcrypto < 2",
     "label-studio-sdk < 1",
-    "oracledb < 2",
+    "oracledb < 3",
```

Comment:
Unrelated - just noticed that they released a new major version, dropping support for python 3.6